### ISSUE TITLE
Feature request: Presentation mode

### ISSUE THREAD

**Issue by gingters** [microsoft/terminal#4176]
# Description of the new feature/enhancement

It would be great if Terminal provides a feature to switch to a "presentation mode".

Presentation mode means:
- Switch to a configurable font size
- Switch to a configurable color scheme (foreground / background color) for better contrast

The presentation mode should toggle to the better contrast / larger font and also be able to toggle back to the previous profile settings.

# Additional info:
I really like to give talks, and if I happen to have some things I need to show on console, I'd love to use Terminal. However, I don't want to add new additional profiles just for presentations, and then copy the presentation profiles over to different shells (which would in fact double my profile count). Also, if I want to change stuff on the presentation profiles, I'd have to do every change for every presentation profile.

So adding separate presentation profiles is just a hacky workaround that involves a lot of work.

**Comment by mkitzan** [comment]
By my understanding a generic version of this feature request would be to have toggle-able profile switching. This would allow you to define your own "presentation" profile which could be switched to without opening/editing the "profiles.json" using a short cut. 

This request dovetails with #4066, where a toggle-able dark/light mode was requested. The same generic feature would cover both requests.

**Comment by egmontkob** [comment]
> toggle-able profile switching

I fully agree. (Is there no issue opened for this yet?) A generic solution sounds much better to me than one-offs like special cases for light/dark or presentation mode.

Many other terminals allow to switch the profile run-time using a menu entry. Obviously some aspects of the profile, like the initial directory or command are ignored then, but others like font size and color palette are adjusted.

**Comment by mkitzan** [comment]
I can't find an open feature request for the generic solutions, but @DHowett-MSFT has triaged #4066 into the spec tracker. 

I did a quick search and found another instance, #3687, which the generic feature could potentially cover. I'll do a deeper search and find all matching instances, before opening a feature request for the generic solution. That comprehensive feature request could then go through the spec process to cover all the related feature requests.

**Comment by saitonakamura** [comment]
@gingters you can do this via nested commands https://docs.microsoft.com/en-us/windows/terminal/command-palette#nested-commands
You can either assign a keybinding or use a command palette (by giving it a proper `"name"`)
